scripts/deployment-helpers.sh

```
@@ -159,3 +206,105 @@ wait_for_csv() {
 
+# Helper function to wait for pods in a namespace to be ready
+wait_for_pods() {
+  local namespace="$1"
+  local timeout="${2:-120}"
+  
+  kubectl get namespace "$namespace" &>/dev/null || return 0
+  
+  echo "⏳ Waiting for pods in $namespace to be ready..."
+  local end=$((SECONDS + timeout))
+  while [ $SECONDS -lt $end ]; do
+    local not_ready=$(kubectl get pods -n "$namespace" --no-headers 2>/dev/null | grep -v -E 'Running|Completed|Succeeded' | wc -l)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)